mdx-models/src/main/java/com/mx/path/model/mdx/accessor/feature/FeatureException.java

```diff
@@ -0,0 +1,45 @@
+package com.mx.path.model.mdx.accessor.feature;
+
+import com.mx.path.core.common.exception.PathRequestException;
+import com.mx.path.core.common.request.Feature;
+
+/**
+ * Base class for exceptions related to features in the path-mdx-model project.
+ * <p>
+ * This exception serves as a foundation for defining feature-specific errors.
+ * It sets appropriate headers for responses, including the feature name and
+ * error descriptor.
+ * <p>
+ * Usage:
+ * <p>
+ * To create a new feature-specific exception, extend this class and implement
+ * the {@code getFeatureName} method to return the corresponding feature name.
+ * Use the constructor to set the user message and error descriptor.
+ */
+public abstract class FeatureException extends PathRequestException {
+
+  private final String feature;
+  private final String errorDescriptor;
```

Comment:
Same with this. Just store as an enum and convert to a string when placing in header.

```suggestion
  private final ErrorDescriptor errorDescriptor;
```